sdk/communication/azure-communication-chat/azure/communication/chat/_shared/user_credential.py

```diff
@@ -21,8 +22,9 @@ class CommunicationTokenCredential(object):
     ON_DEMAND_REFRESHING_INTERVAL_MINUTES = 2
 
     def __init__(self,
-            communication_token_refresh_options
+            token # type: str
```

Comment:
Do we need to add `**kwargs` here to support passing in the "token refresher"?
Or is this not really needed for now?

Reply:
@annatisch The token refresher has just the two fields curently. Don't think we need **kwargs. What is the recommended approach though?